toolkits/landcover/impl/dataset.js

```diff
@@ -168,12 +189,12 @@ Dataset.prototype.addBandIndices = function(var_names) {
   var indices = Array.prototype.slice.call(arguments);
 
   // Compute the transformation to common band names.
-  var bandNames = this.collection_.first().bandNames();
-  var commonNames = this.computeCommonBandNames_(bandNames);
+  var dstBands = this.getCommonBandNames();
+  var srcBands = this.lookupCommonBandNames(dstBands, this.collection_.first().bandNames());
```

Comment:
Should `getCommandBandNames()` and `lookupCommonBandNames()` be private? If we have any doubt that they'll be needed then they should be to reduce API surface area.

Reply:
Ack.